src/server.cpp

```diff
@@ -1135,6 +1181,13 @@ void CServer::MixEncodeTransmitData ( const int iChanCnt,
         // Mono target channel -------------------------------------------------
         for ( j = 0; j < iNumClients; j++ )
         {
+            // check if client is blocked.
+            // one can always hear himself --> if we currently iterate through the blocked client we do send him audio
+            if ( ( j != iCurChanID && vecChannels[j].IsBlocked() ) || ( j != iCurChanID && currentClientBlocked ) )
+            {
```

Comment:
This is the interesting part: 

In short: 

If you're blocked, nobody can hear you and you can not be heared. You will only be able to hear yourself. This makes it possible to have real private sessions in terms of "although you're connected to a server, nobody can come into your rehearsal"